config/README.md

```diff
@@ -27,6 +27,19 @@ locations, in order:
 If no configuration file is found in these locations, **iceoryx2** will use
 its default settings.
 
+## Note
```

Comment:
Since this is a breaking change could you please also add the issue to the release notes in `doc/release-notes/iceoryx2-unreleased.md`
and add an additional breaking change not at the end of the file.